.github/workflows/linux.yml

```
@@ -30,7 +30,43 @@ env:
   GENAI_SAMPLES_NAME: genai_samples.tar.gz
 
 jobs:
+  Smart_CI:
+    runs-on: ubuntu-latest
+    outputs:
+      affected_components: "${{ steps.smart_ci.outputs.affected_components }}"
+      changed_components: "${{ steps.smart_ci.outputs.changed_components }}"
+      skip_workflow: "${{ steps.smart_ci.outputs.skip_workflow }}"
+    steps:
+      - name: checkout action
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          repository: 'openvinotoolkit/openvino'
+          path: 'openvino'
+          ref: ${{ env.OV_BRANCH }}
+          sparse-checkout: .github/actions/smart-ci
+
+      - name: Get affected components
+        id: smart_ci
+        uses: ./.github/actions/smart-ci
```

Since you're cloning to "openvino" path I think it will be something like
```suggestion
        uses: ./openvino/.github/actions/smart-ci
```